vault/external_tests/raft/raft_test.go

```diff
@@ -40,10 +41,15 @@ type RaftClusterOpts struct {
 	InmemCluster                   bool
 	EnableAutopilot                bool
 	PhysicalFactoryConfig          map[string]interface{}
+	PhysicalFactory                func(t testingintf.T, coreIdx int, logger log.Logger, conf map[string]interface{}) *vault.PhysicalBackendBundle
 	DisablePerfStandby             bool
 	EnableResponseHeaderRaftNodeID bool
 	NumCores                       int
 	Seal                           vault.Seal
+	VersionMap                     map[int]string
+	RedundancyZoneMap              map[int]string
+	AutopilotReconcileInterval     time.Duration
```

Comment:
I don't see this field or the following used in raftCluster(), am I missing something?

Reply:
They're used in the ent tests.

Reply:
I couldn't find any reference to them on ent.  I suspect your PhysicalFactoryConfig has superseded them.

Reply:
Ah yes, so it did. I'll remove them.